web/packages/teleport/src/Main/Main.tsx

```diff
@@ -339,6 +339,7 @@ export const ContentMinWidth = ({ children }: { children: ReactNode }) => {
           flex-direction: column;
           flex: 1;
           ${enforceMinWidth ? 'min-width: 1000px;' : ''}
+          min-height: 0;
```

Comment:
What's the reason for this specific change? I've just got a PR to review which fixes a problem that stems from each page of Web UI being wrapped in a flex parent with `min-height: 0`. https://github.com/gravitational/teleport.e/pull/5897

Reply:
With `min-height: 0`, the flex layout for TAG correctly has the sidebar scrolling - 

<img width="1912" alt="image" src="https://github.com/user-attachments/assets/451b4924-33d6-477b-9f73-3aefba63a97d" />

If it's removed, the whole page scrolls instead

<img width="1912" alt="image" src="https://github.com/user-attachments/assets/726e3212-f3f7-43ac-ac10-e802ce1e52fe" />

Reply:
Thanks. I suppose maybe this could be overrideable on a per page basis like `enforceMinWidth`? That sounds like something we should address just before the test plan and not now though.

Reply:
Yeah that is an option. I wasn't aware that `min-height: 0` had side effects outside of making flex children not overflow the parent